packages/components/src/Switch/Switch.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { LegacyRef, RefObject, useRef } from 'react';
+import { useFocusRing } from '@react-aria/focus';
+import { useSwitch } from '@react-aria/switch';
+import { VisuallyHidden } from '@react-aria/visually-hidden';
+import { useToggleState } from '@react-stately/toggle';
+import { ToggleProps } from '@react-types/checkbox';
+import { AriaSwitchProps } from '@react-types/switch';
+
+import { ComponentProps } from '@marigold/types';
+import { conditional } from '@marigold/system';
+
+import { Box } from '../Box';
+import { Label } from '../Label';
+
+// Theme Extension
+// ---------------
+export interface SwitchThemeExtension<Value> {
+  switch?: {
+    [key: string]: Value;
+  };
+}
+
+// Props
+// ---------------
+export type SwitchProps = {
+  variant?: string;
+  labelVariant?: string;
+  disabled?: boolean;
+} & AriaSwitchProps &
+  ToggleProps &
+  ComponentProps<'input'>;
+
+// Component
+// ---------------
+export const Switch: React.FC<SwitchProps> = ({
+  variant = '',
+  labelVariant = 'above',
```

Comment:
Can we use a child like in `<Checkbox>`?

Reply:
Yes I do it with props.children but I can also change to children